src/compiler/Restler.Compiler/Annotations.fs

```diff
@@ -220,4 +221,70 @@ let getGlobalAnnotationsFromFile filePath =
             printfn "ERROR: invalid annotation file: x-restler-global-annotations must be the key"
             raise (ArgumentException("invalid annotation file"))
     else
-        List.empty
\ No newline at end of file
+        List.empty
+
+// Gets the REST-ler dependency annotation from OpenAPI v3 links
```

Comment:
>REST-ler [](http://example.com/codeflow?start=12&length=8)

RESTler (without dash).  Very early versions did use REST-ler, feel free to delete dashes if you ever see them in the code base